cart.html

```diff
@@ -27,13 +26,17 @@
 
 				<div class="text-white flex items-center text-sm space-x-6s mx-6 ml-auto">
 					<div class="link white pointer mx-2">
-						<p>Sign in</p>
-						<p class="font-bold text-sm">Account & Lists</p>
+						<a href="./login.html">
+							<p>Sign in</p>
+							<p class="font-bold text-sm">Account & Lists</p>
+						</a>
 					</div>
 
 					<div class="link white pointer mx-2">
-						<p>Wishlist	</p>
-						<p class="font-bold text-sm">& Orders</p>
+						<a href="./wishlist.html">
+							<p>Wishlist	</p>
```

Comment:
Suggestion: Please use a tool for code formatting. This will take care of the extra spacing too.